posthog/temporal/data_imports/external_data_job.py

```diff
@@ -260,12 +262,24 @@ async def run(self, inputs: ExternalDataWorkflowInputs):
                 else {"start_to_close_timeout": dt.timedelta(hours=12), "retry_policy": RetryPolicy(maximum_attempts=3)}
             )
 
-            await workflow.execute_activity(
-                import_data_activity,
-                job_inputs,
-                heartbeat_timeout=dt.timedelta(minutes=5),
-                **timeout_params,
-            )  # type: ignore
+            if is_posthog_team(inputs.team_id) and (
+                source_type == ExternalDataSource.Type.POSTGRES or source_type == ExternalDataSource.Type.BIGQUERY
+            ):
+                # Sync activity for testing
+                await workflow.execute_activity(
+                    import_data_activity_sync,
+                    job_inputs,
+                    heartbeat_timeout=dt.timedelta(minutes=5),
```

Comment:
Heartbeating needs to be implemented for this timeout to make sense. I don't think the new sync activity has any heartbeating (yet).